docs/product/issues/issue-details/replay-issues/hydration-error.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Hydration Error
+sidebar_order: 50
+description: "Learn about hydration errors."
```

Comment:
```suggestion
description: "Learn about Session Replay hydration errors."
```

Reply:
to match the description on https://docs.sentry.io/product/issues/issue-details/replay-issues/